crates/oxc_ecmascript/Cargo.toml

```diff
@@ -28,6 +28,10 @@ oxc_syntax = { workspace = true, features = ["to_js_string"] }
 num-bigint = { workspace = true }
 num-traits = { workspace = true }
 
+[dev-dependencies]
+oxc_allocator = { workspace = true }
+oxc_parser = { workspace = true }
```

Comment:
This creates a cyclic dependency with `oxc_parser` (an unfortunate cargo limitation). This is why all the tests are in `oxc_minifier` :-(

Reply:
The cyclic dependency breaks release, but nothing happens during development :thrug:

Reply:
Ah, I see. I was wondering why this wasn't done as it worked. 😅
Let me move the tests.